src/Rule/MinLength.php

```
@@ -0,0 +1,53 @@
+<?php
+
+namespace Morebec\Validator\Rule;
+
+
+use Morebec\Validator\ValidationRuleInterface;
+
+class MinLength implements ValidationRuleInterface
+{
+    /**
+     * @var int
+     */
+    private $minLength;
+
+    /**
+     * @var string|null
+     */
+    private $message;
+
+    /**
+     * MinLength constructor.
+     * @param int $minLength
+     * @param string|null $message
+     */
+    public function __construct(
+        int $minLength,
+        ?string $message = null
+    )
+    {
+        $this->minLength = $minLength;
```

Could you validate that the parameter passed is a positive integer or zero?
If a user of the function passes a negative number, this would cause the rule to always true.
It would be better to throw an InvalidArgumentException.